SPECS/curl.spec

```diff
@@ -322,7 +309,7 @@ rm -f ${RPM_BUILD_ROOT}%{_libdir}/libcurl.la
 # should be fixed in next release https://github.com/curl/curl/pull/12843
 rm -f ${RPM_BUILD_ROOT}%{_mandir}/man1/mk-ca-bundle.1*
 
-%ldconfig_scriptlets -n libcurl
+/sbin/ldconfig -n libcurl
```

Comment:
Where does this come from?

Reply:
Because it failed if we didn't do like this. @tescande helped me on this one.

Reply:
Well, it's wrong :)

Reply:
And do you have any suggestion on this change as the initial line doesn't work?

Reply:
Yes, but I would prefer not to be the one who explains why it's wrong to replace this macro without knowing what it used to do.

Reply:
And `/sbin/ldconfig -n libcurl` clearly makes no sense.

Reply:
Check what the older curl SRPM did with ldconfig...

Reply:
Ok, so my bad. I thought the install steps needed to refresh its so cache before continuing the installation.

So the solution is to replace the macro `%ldconfig_scriptlets` with:
```
%post -n libcurl -p /sbin/ldconfig
%postun -p /sbin/ldconfig
```

Reply:
I allready changed it with the entry in the older srpm from koji.